app/assets/javascripts/admin/addon/components/admin-user-field-item.js

```diff
@@ -79,6 +102,17 @@ export default Component.extend(bufferedProperty("userField"), {
       ...this.adminCustomUserFields.additionalProperties
     );
 
+    let confirm = true;
+
+    if (attrs.requirement === "for_all_users") {
+      confirm = await this._confirmChanges();
+    }
+
+    if (!confirm) {
+      this.cancel();
```

Comment:
I don't think we need this, it resets the whole form. We can just return, then they can make tweaks to the requirements and re-submit